src/include/migraphx/module.hpp

```diff
@@ -36,6 +39,8 @@ struct module
     // copy constructor
     module(const module&);
 
+    module(const module&, std::unordered_map<instruction_ref, instruction_ref>&);
```

Comment:
Is this constructor used anywhere?

Reply:
It is not used. Actually, there is no implementation for this function at all. Just removed it.